README.md

```diff
@@ -14,6 +14,7 @@ In your `build.gradle`:
  dependencies {
    debugCompile 'com.squareup.leakcanary:leakcanary-android:1.3.1'
    releaseCompile 'com.squareup.leakcanary:leakcanary-android-no-op:1.3.1'
+   testCompile 'com.squareup.leakcanary:leakcanary-android-no-op:1.3.1'	//Optional: to disable in unit test
```

Comment:
Can you remove the `//Optional: to disable in unit test` comment? I expect people to be smart enough to know what their build look like and remove it if needed